examples/all-clusters-app/all-clusters-common/all-clusters-app.zap

```diff
@@ -1677,6 +1677,72 @@
             }
           ]
         },
+        {
+          "name": "Localization Time Format",
+          "code": 44,
+          "mfgCode": null,
+          "define": "LOCALIZATION_TIME_FORMAT_CLUSTER",
+          "side": "client",
+          "enabled": 0,
+          "commands": [],
+          "attributes": []
+        },
+        {
+          "name": "Localization Time Format",
+          "code": 44,
+          "mfgCode": null,
+          "define": "LOCALIZATION_TIME_FORMAT_CLUSTER",
+          "side": "server",
+          "enabled": 1,
+          "commands": [],
+          "attributes": [
+            {
+              "name": "HourFormat",
+              "code": 0,
+              "mfgCode": null,
+              "side": "server",
+              "included": 1,
+              "storageOption": "RAM",
```

Comment:
This should be stored in NVM, not RAM, I'd think....  File spec issues as needed?